File: 3. 반복문/1110.py
result = input().zfill(2)
num = list(map(int, str(result)))
cnt = 0

while True:
  num.append(int(str(num[0] + num[1])[-1]))
  num.pop(0)
  cnt += 1

  if str(num[0]) + str(num[1]) == result:
    break

print(cnt)

File: C++/3. 반복문/1110.cpp
#include <iostream>
using namespace std;

int main() {
	int n, cnt = 0, num, temp;
	scanf("%d", &n);
	num=n;

	while (1){
		temp=(n%10)*10+(n/10+n%10)%10;
		cnt++;

		n=temp;
		if(temp==num)
			break;
	}
	printf("%d", cnt);
}